### ISSUE TITLE
Remove instructions for dependencies, or clarify instructions for using?

### ISSUE THREAD

**Issue by jrhorn424** [ga-wdi-boston/browser-template#75]
Developers in 016 thought they had to update their dependencies (great job reading!).

I had intended this as a note for consultants. The instructions for using the template is intended for developers.

I think it's a bad idea to have developers update their dependencies to versions outside what we've specified ourselves in the template.

**Comment by gaand** [comment]
Agreed on not having them update dependencies.  Should we have a separate file for these or perhaps capture them elsewhere (e.g in a general "update template" section of the wiki)?
